operations/bills.md

```diff
@@ -107,7 +107,8 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
                 "Value": {
                     "ItalianFiscalCode": null
                 }
-            }
+            },
+            "EnterpriseData": null
```

Comment:
Include also data that actually make the enterprise data, so that people know what to expect when EnterpriseData returns